### ISSUE TITLE
Feature importances: show per target

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#120]
For trees and tree ensembles that perform multiclass classification or multitarget regression it's maybe more informative to view feature importances per target. Maybe this should be optional.
Related sklearn issue: https://github.com/scikit-learn/scikit-learn/issues/8055
For xgboost, feature importances are parsed from get_dump, so it's easy to separate trees for different targets.

**Comment by kmike** [comment]
A related ticket is https://github.com/TeamHG-Memex/eli5/issues/56; the description was unclear, but what I meant is something opposite of this ticket: provide an option to show features for all targets in a single combined table, without splitting them by target, similar to how feature importances work now.
